jena-arq/src/main/java/org/apache/jena/riot/lang/rdfxml/SysRRX.java

```diff
@@ -29,9 +29,22 @@
  */
 public class SysRRX {
 
+    /**
+     * Creates and initializes a javax.xml.stream.XMLInputFactory#newDefaultFactory().
+     * @return
+     */
     public static XMLInputFactory createXMLInputFactory() {
-        XMLInputFactory xmlInputFactory = XMLInputFactory.newInstance();
-        JenaXMLInput.initXMLInputFactory(xmlInputFactory);
+        return initAndConfigure(XMLInputFactory.newInstance());
+    }
+
+    /**
+     * Configures the parser to be safe and sets necessary properties.
+     * This method should be called when another factory than javax.xml.stream.XMLInputFactory#newDefaultFactory()
+     * shall be used.
+     * @param xmlInputFactory
+     */
+    public static <E extends XMLInputFactory> E initAndConfigure(final E xmlInputFactory) {
+         JenaXMLInput.initXMLInputFactory(xmlInputFactory);
```

Comment:
```suggestion
        JenaXMLInput.initXMLInputFactory(xmlInputFactory);
```
Misaligned indentation